appveyor.yml

```diff
@@ -2,39 +2,33 @@ version: "{build}"
 
 cache:
  - C:\Cache
- - C:\gopath\pkg\dep\sources -> Gopkg.lock
+ - C:\gopath/pkg/mod -> go.sum
 
 clone_folder: C:\gopath\src\github.com\influxdata\telegraf
 
 environment:
   GOPATH: C:\gopath
+  GO111MODULE: 'on'
 
 platform: x64
 
 install:
   - IF NOT EXIST "C:\Cache" mkdir C:\Cache
   - IF NOT EXIST "C:\Cache\go1.12.5.msi" curl -o "C:\Cache\go1.12.5.msi" https://storage.googleapis.com/golang/go1.12.5.windows-amd64.msi
-  - IF NOT EXIST "C:\Cache\gnuwin32-bin.zip" curl -o "C:\Cache\gnuwin32-bin.zip" https://dl.influxdata.com/telegraf/ci/make-3.81-bin.zip
-  - IF NOT EXIST "C:\Cache\gnuwin32-dep.zip" curl -o "C:\Cache\gnuwin32-dep.zip" https://dl.influxdata.com/telegraf/ci/make-3.81-dep.zip
   - IF EXIST "C:\Go" rmdir /S /Q C:\Go
   - msiexec.exe /i "C:\Cache\go1.12.5.msi" /quiet
-  - 7z x "C:\Cache\gnuwin32-bin.zip" -oC:\GnuWin32 -y
-  - 7z x "C:\Cache\gnuwin32-dep.zip" -oC:\GnuWin32 -y
-  - go get -d github.com/golang/dep
-  - cd "%GOPATH%\src\github.com\golang\dep"
-  - git checkout -q v0.5.0
-  - go install -ldflags="-X main.version=v0.5.0" ./cmd/dep
+  - choco install make
```

Comment:
Nice, but do you know if this gets cached?  If so, and especially if we can install specific versions, maybe we should use it for Go too?

Reply:
Im not sure how to get appveyors cache to speed up chocolatey installs. Though I watched a fair few appveyor builds while getting this to work and it took no time at all. Also I had issues with the old gnu make installation. It just kept failing with a non-descript error code. As soon as I installed it via choco it just worked :/

Reply:
Just had a look at there is a `choco install golang` 👍

Reply:
Looks like we could cache the directory chocolately uses to download packages into https://help.appveyor.com/discussions/suggestions/592-speeding-up-chocolatey-downloads-and-installs and then invalidate it on appveyor change. We could then explicitly set golang version?
Thoughts?

Reply:
Yeah let's try it